lib/dok_matrix.rb

```diff
@@ -52,13 +52,6 @@ def []=(row, col, value)
   alias set_element []=
   alias set_component []=
 
-  private def set_value(row, col, value)
-    raise(ArgumentError) unless row >= 0 && col >= 0
-    raise(TypeError) unless value.is_a?(Numeric) || value.nil?
-
-    @dict[[row, col]] = value
-  end
-
   def **(other)
     raise(TypeError) unless other.is_a?(Numeric)
 
```

Comment:
I believe the line below should read 'super other'? Why is it 'super**other'

Reply:
Ya you correct. Fixing